<ui version="4.0" >
 <class>FtpUploadOptsForm</class>
 <widget class="QWidget" name="FtpUploadOptsForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QLabel" name="labelUrl" >
     <property name="text" >
      <string>Target URL:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="lineTarget" />
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>FTP connection mode:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="comboFtpMode" />
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Authentication</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>9</number>
      </property>
      <property name="topMargin" >
       <number>9</number>
      </property>
      <property name="rightMargin" >
       <number>9</number>
      </property>
      <property name="bottomMargin" >
       <number>9</number>
      </property>
      <property name="horizontalSpacing" >
       <number>6</number>
      </property>
      <property name="verticalSpacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="linePassword" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="lineUsername" />
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Username:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Proxy server:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QComboBox" name="comboProxy" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_7" >
     <property name="text" >
      <string>Bind IP address:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="lineAddrBind" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineTarget</tabstop>
  <tabstop>comboFtpMode</tabstop>
  <tabstop>lineUsername</tabstop>
  <tabstop>linePassword</tabstop>
  <tabstop>comboProxy</tabstop>
  <tabstop>lineAddrBind</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
